tests/testRandom1.py

```diff
@@ -139,7 +139,7 @@ def testRandomGaussianImage(self):
     def testRandomChisqImage(self):
         nu = 10
         afwMath.randomChisqImage(self.image, self.rand, nu)
-        stats = afwMath.makeStatistics(self.image, afwMath.MEAN | afwMath.VARIANCE)
+        stats = afwMath.makeStatistics(self.image, int(afwMath.MEAN) | int(afwMath.VARIANCE))
```

Comment:
Yes, and no. Yes it was needed at the time. But now pybind11 supports the `py::is_arithmetic()` tag to `py::enum_` which adds binary operators and removes the need for the cast in these cases. All enums that are to be compared as such should get this tag. I changed it here now.